### ISSUE TITLE
Error with cosmosis postprocessing 

### ISSUE THREAD

**Issue by audita-nimas** [joezuntz/cosmosis#142]
Hi Joe,

I got the trouble when I tried this:cosmosis-postprocess output.txt --no-plots -o outputfolder --getdist --derive=derive_s8.py. this is my error:
```
Deriving new columns from these functions in derive_s8.py:
Removed no burn in
Optimization terminated successfully.
         Current function value: -0.999991
         Iterations: 13
         Function evaluations: 26
Failed in one of the postprocessing steps:  <cosmosis.postprocessing.statistics.PolychordStatistics object at 0x13d2a5720>
Here is the error stack:
Traceback (most recent call last):
  File "/Users/nemas/cosmosis-trial8/env/lib/python3.10/site-packages/cosmosis/postprocessing/postprocess_base.py", line 220, in run
    files += e.run()
  File "/Users/nemas/cosmosis-trial8/env/lib/python3.10/site-packages/cosmosis/postprocessing/statistics.py", line 718, in run
    files = super(MultinestStatistics,self).run()
  File "/Users/nemas/cosmosis-trial8/env/lib/python3.10/site-packages/cosmosis/postprocessing/statistics.py", line 352, in run
    N = self.compute_basic_statsgd()
  File "/Users/nemas/cosmosis-trial8/env/lib/python3.10/site-packages/cosmosis/postprocessing/statistics.py", line 344, in compute_basic_statsgd
    self.uerr95.append(col_results[9])
IndexError: list index out of range
```
do you know what happened?

**Comment by joezuntz** [comment]
Thanks for spotting this mistake - I am testing a fix now and will push a new release when the tests complete.

**Comment by joezuntz** [comment]
This is fixed in cosmosis v3.12.1, which is available on pypi and conda.  Thanks for finding this!
